### PR TITLE
[NUI] fix svace issue for texture

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
fix svace issue for texture

### CODE REVIEW

**Review by huayongxu — APPROVED** [review]
lgtm.

File: src/Tizen.NUI/src/internal/FrameBroker/FrameBrokerBase.cs [hunk 1]
```diff
@@ -364,7 +364,7 @@ internal Renderer GetRenderer(FrameData data)
                     break;
             }
 
-            texture.Dispose();
+            texture?.Dispose();
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/FrameBroker/FrameBrokerBase.cs:367]: How about geometry and shader ? also don't they need null check?

Comment (zhouleonlei) [reply src/Tizen.NUI/src/internal/FrameBroker/FrameBrokerBase.cs:367]: > How about geometry and shader ? also don't they need null check?

They have been New in 
            Geometry geometry = CreateQuadGeometry();
            Shader shader = CreateShader();

And in the Switch case, function return Null directly:
            switch (data.Type)
            {
                case FrameData.FrameType.RemoteSurfaceTbmSurface:
                    if (data.TbmSurface == null)
                    {
                        geometry.Dispose();
                        shader.Dispose();
                        return null;
                    }

**Review by AdunFang — COMMENTED** [review]
It is OK.

**Review by elishateng — COMMENTED** [review]
LGTM

**Pull request merged**